slackclient/_server.py

```diff
@@ -26,16 +26,11 @@ def __init__(self, token, connect=True):
             self.rtm_connect()
 
     def __eq__(self, compare_str):
-        if compare_str == self.domain or compare_str == self.token:
-            return True
-        else:
-            return False
+        return compare_str in (self.domain, self.token)
```

Comment:
I feel like this combines too many operations into a single line. I'd prefer to keep the return True, else return False part of this logic.